modules/oe_theme_content_organisation/oe_theme_content_organisation.info.yml

```diff
@@ -5,6 +5,7 @@ package: OpenEuropa
 core: 8.x
 dependencies:
   - oe_content:oe_content_organisation
+  - oe_content:oe_content_organisation_person_reference
```

Comment:
We need to enable this via a hook update now, since we removed it from the oe_content_organisation hook update / dependency.